mne/viz/_3d.py

```diff
@@ -52,6 +54,97 @@ def _fiducial_coords(points, coord_frame=None):
     return np.array([points_[i]['r'] for i in FIDUCIAL_ORDER])
 
 
+def plot_head_positions(pos, mode='traces', cmap='viridis', direction='z',
+                        show=True):
+    """Plot head positions.
+
+    Parameters
+    ----------
+    pos : ndarray, shape (n_pos, 10)
+        The head position data.
+    mode : str
+        Can be 'traces' (default) to show position and quaternion traces,
+        or 'field' to show the position as a vector field over time.
+        The 'field' mode requires matplotlib 1.4+.
+    cmap : matplotlib Colormap
+        Colormap to use for the trace plot, default is "viridis".
+    direction : str
+        Can be any combination of "x", "y", or "z" (default: "z") to show
+        directional axes in "field" mode.
+    show : bool
+        Show figure if True. Defaults to True.
+
+    Returns
+    -------
+    fig : Instance of matplotlib.figure.Figure
+        The figure.
+    """
+    from ..chpi import head_pos_to_trans_rot_t
+    import matplotlib.pyplot as plt
+    from matplotlib.colors import Normalize
+    from mpl_toolkits.mplot3d.art3d import Line3DCollection
+    from mpl_toolkits.mplot3d import axes3d  # noqa: F401
+    if not isinstance(mode, string_types) or mode not in ('traces', 'field'):
+        raise ValueError('mode must be "traces" or "field", got %s' % (mode,))
+    trans, rot, t = head_pos_to_trans_rot_t(pos)  # also ensures pos is okay
+    # trans, rot, and t are for dev_head_t, but what we really want
+    # is head_dev_t (i.e., where the head origin is in device coords)
+    use_trans = np.einsum('ijk,ik->ij', rot[:, :3, :3].transpose([0, 2, 1]),
+                          -trans) * 1000
+    use_rot = rot.transpose([0, 2, 1])
+    use_quats = -pos[:, 1:4]  # inverse (like doing rot.T)
+    if isinstance(cmap, string_types) and cmap == 'viridis' and \
+            not check_version('matplotlib', '1.5'):
```

Comment:
``if cmap == 'viridis' and not check_version('matplotlib', '1.5'):`` should be enough.